src/chart/context.rs

```diff
@@ -335,17 +342,25 @@ impl<'a, DB: DrawingBackend, X: Ranged, Y: Ranged> ChartContext<'a, DB, RangedCo
                 };
 
                 area.draw_text(&t, label_style, (text_x, text_y))?;
-
+                // TODO: need some adjustments here
```

Comment:
Actually I am thinking about do the both thing:
1. provide an API for tick_size adjustment 
2. supports negative tick_size
And I am not sure if we can unify the implementation for the inward one.

Reply:
This part of code i was most confused here i need just mirror all Path Lines coordinates.
Make tw/th = 0 and 0 = tw/th. I faced with some problems when realized there is truncate function. Its good if you know how to make it better.